examples/dynamic-projections-dask.py

```diff
@@ -0,0 +1,58 @@
+"""
+Using dask array operations, one can dynamically take arbitrary slices
+and computations of a source dask array and display the results in napari.
+When the computation takes one or more parameters, one can tie a UI to
+them using magicgui.
+"""
+
+import napari
+import dask.array as da
+from dask.array.lib.stride_tricks import sliding_window_view
+from skimage import data
+
+##############################################################################
+# Part 1: using code to view a specific value.
+
+blobs = data.binary_blobs(length=64, n_dim=3)
+blobs_dask = da.from_array(blobs, chunks=(1, 64, 64))
+
+# original shape [60, 1, 1, 5, 64, 64], slice out singleton axes
+blobs_dask_windows = sliding_window_view(
+    blobs_dask, window_shape=(5, 64, 64)
+)[:, 0, 0, ...]  
```

Comment:
Wouldn't a "squeeze" be clearer?

Reply:
squeeze might remove the sliding window axis if its size is 1 (as in the second example, at the start). That's the reason why I generally dislike using squeeze.

... But, I just saw that since NumPy 1.7, squeeze supports an `axis=` argument — I'd be happy using that!